_docs/concepts/configuration/space.md

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
Looking at the words in the glossary, I don't think these should be in there. They're all too specific terms to be in the general platform

Reply:
Do you suggest to put the glossary under config concepts? I prefer not. I'd like to have one "contains all" Index like experience for glossary. Most ppl use glossary like dictionary, they don't really care what other terms are there too.

Reply:
I agree with Martin, I don't think we need all of this in the glossary.

I'm not sure we need it on the website at all.

What are you trying to accomplish by adding all of this to the website?  What is the goal?

For example, having "X config" entries for every type of configuration in the glossary is a waste. People know what configuration means.

So for example, rather than defining "API Management config" as "Configuration for API Management", we should have an entry for "API Management" that defines what we mean by API Management.

The config entries are just redundant.